src/main/java/no/nav/brregstub/mapper/RolleoversiktMapper.java

```diff
@@ -69,6 +85,33 @@ private static ResponseHeader mapTilResponseHeader(RolleoversiktTo to) {
         return responseHeader;
     }
 
+    private static ResponseHeader mapTilResponseHeader(RsRolleoversikt rsRolleoversikt) {
+        var responseHeader = new ResponseHeader();
+        responseHeader.setProssessDato(localDateToXmlGregorianCalendar(LocalDate.now()));
+        responseHeader.setTjeneste(TJENESTE_NAVN);
+        responseHeader.setFodselsnr(rsRolleoversikt.getFnr());
+        responseHeader.setHovedStatus(rsRolleoversikt.getHovedstatus());
+        var underStatus = new UnderStatus();
+        if (rsRolleoversikt.getUnderstatuser() == null || rsRolleoversikt.getUnderstatuser().isEmpty()) {
```

Comment:
Jeg liker veldig godt det mønsteret med eksplisitte gettere for collection (etter behov). I klasse RsRolleoversikt lager du 
```
public List<Integer> getUnderStatuser() {
    if (underStatuser == null) {
        underStatuser = new ArrayList<>();
    }
   return understatuser;
```
og får penere kode der underStatuser gettes:
```
if (rsRolleoversikt.getUnderstatuser().isEmpty()) {
```